Documentation/gettingstarted/hubble-configuration.rst

```diff
@@ -24,212 +24,272 @@ This allows Hubble Relay to communicate with all Hubble instances in the
-
-This method relies on `cert-manager <https://cert-manager.io/>`__ to generate
-the TLS certificates. cert-manager has becomes the de facto way to manage TLS on
-Kubernetes, and it has the following advantages compared to the previously
-documented methods:
-
-* Support multiple issuers (e.g. a custom CA,
-  `Vault <https://www.vaultproject.io/>`__,
-  `Let's Encrypt <https://letsencrypt.org/>`__,
-  `Google's Certificate Authority Service <https://cloud.google.com/certificate-authority-service>`__,
-  and more) allowing to choose the issuer fitting your organization's
-  requirements.
-* Manages certificates via a
-  `CRD <https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/>`__
-  which is easier to inspect with Kubernetes tools than PEM file.
-
-**Installation steps**:
-
-#. First, install `cert-manager <https://cert-manager.io/docs/installation/>`__
-   and setup an `issuer <https://cert-manager.io/docs/configuration/>`_.
-   Please make sure that your issuer is able to create certificates under the
-   ``cilium.io`` domain name.
-#. Install/upgrade Cilium including the following Helm flags:
+    .. group-tab:: CronJob (certgen)
 
-::
+        When using certgen, TLS certificates are generated at installation time
+        and a Kubernetes ``CronJob`` is scheduled to renew them (regardless of
+        their expiration date).
 
-    --set hubble.tls.auto.enabled=true               # enable automatic TLS certificate generation
-    --set hubble.tls.auto.method=certmanager         # auto generate certificates using cert-manager
-    --set hubble.tls.auto.certValidityDuration=1095  # certificates validity duration in days (default 3 years)
-    --set hubble.tls.auto.certManagerIssuerRef.group="cert-manager.io" # Reference to cert-manager's issuer
-    --set hubble.tls.auto.certManagerIssuerRef.kind="ClusterIssuer"
-    --set hubble.tls.auto.certManagerIssuerRef.name="ca-issuer"
+        ::
 
-**Troubleshooting**:
+            --set hubble.tls.auto.enabled=true               # enable automatic TLS certificate generation
+            --set hubble.tls.auto.method=cronJob             # auto generate certificates using cronJob method
+            --set hubble.tls.auto.certValidityDuration=1095  # certificates validity duration in days (default 3 years)
+            --set hubble.tls.auto.schedule="0 0 1 */4 *"     # schedule for certificates re-generation (crontab syntax)
 
-While installing Cilium or cert-manager you may get the following error:
+    .. group-tab:: Helm
 
-::
+        When using Helm, TLS certificates are (re-)generated every time Helm is used
+        for install or upgrade. As Hubble server and Hubble Relay support TLS
+        certificates hot reloading, including CA certificates, this does not disrupt
+        any existing connection. New connections are automatically established using
+        the new certificates without having to restart Hubble server or Hubble
+        Relay.
```

Comment:
I think it's great to mention that Hubble server and relay support TLS certs hot reloading but it's probably best to call out in a more generic section as this is not specific to the Helm way only but is valid whichever method is used to renew TLS certificates.

Reply:
Hmm I just copied this from the existing docs. I think the reason it's mentioned is because we want to call out that using helm to rotate certificates should not cause disruption.

Reply:
Hmm I can see moving this to the top and rephrasing slightly